octobot_trading/exchanges/connectors/ccxt_exchange.py

```diff
@@ -284,11 +461,26 @@ async def get_recent_trades(self, symbol: str, limit: int = 50, **kwargs: dict)
     async def get_price_ticker(self, symbol: str, **kwargs: dict) -> typing.Optional[dict]:
         try:
             with self.error_describer():
-                return await self.client.fetch_ticker(symbol, params=kwargs)
+                ticker = await self.client.fetch_ticker(symbol, params=kwargs)
+                # some exchanges are missing values
+                if not ticker[ecoc.TIMESTAMP.value]:
+                    ticker[t_cols.TIMESTAMP.value] = self.client.milliseconds()
+                if not ticker[t_cols.BASE_VOLUME.value]:
+                    try:
+                        ticker[t_cols.BASE_VOLUME.value] \
+                            = ticker[t_cols.QUOTE_VOLUME.value] / ticker[t_cols.CLOSE.value]
+                    except Exception as e:
+                        # todo is it critical?
+                        self.logger.warning(f"Failed to get ticker parameter: "
+                                            f"base volume - symbol {symbol} - error: {e}")
+                return ticker
```

Comment:
Regarding CANDLE_LOADING_LIMIT i fixed it and changed it so it'll warn if it will ever happen

I improved ticker parsing see parser/ticker_parser now its properly handling missing values now